Data for running GLM on Cannonsville and Pepacton

aappling-usgs opened the pull request:
This repo collects, subsets, and pushes data relevant to reservoir GLM modeling up to [ScienceBase](https://www.sciencebase.gov/catalog/item/6006eec9d34e592d867201d0). The meteo files are being provided to this repo by Jordan via email. Other files come from lake-temperature-model-prep or delaware-model-prep repos.

aappling-usgs committed:
initial data release (2 reservoirs)

aappling-usgs committed:
add inflow/outflow data & SNTemp predictions

hcorson-dosch-usgs commented on remake.yml:
```diff
@@ -5,45 +5,100 @@ packages:
+      in_repo = I('../lake-temperature-model-prep/'),
+      site_ids = site_ids_drb)
+      
+  # Not quite robust pipelining - if the files in in_data change
```
What does `in_data` refer to here? Files in the `lake-temperature-model-prep` repo?

hcorson-dosch-usgs replied:
Ah I see - looks like that folder is generated by the fetch_meteo_files task table.

aappling-usgs replied:
Worse than that - it's actually a manually created, manually populated folder. (At least, i don't remember / couldn't find a `dir.create('in_data')` call, and I know for sure that I hand-copy the NLDAS meteo files into that folder after receiving them from Jordan via Teams or email.)

hcorson-dosch-usgs reviewed (approved):
Looks good. Unfortunately, I can't test the code by rebuilding, b/c of the implicit target dependencies from other repos, but I traced through the code to build each of the targets, and everything looks good and is clearly commented.

aappling-usgs commented:
Thanks for reviewing, @hcorson-dosch ! Jake, Sam, and I agreed this morning that we'll be going with a policy of nearly always getting a PR review (except for completely trivial PRs) and then having the PR creator merge her/his own once she/he is satisfied that they've addressed the comments. So I'm merging now.
